src/transformers/models/wav2vec2/feature_extraction_wav2vec2.py

```diff
@@ -79,13 +79,20 @@ def __init__(
         self.do_normalize = do_normalize
 
     @staticmethod
-    def zero_mean_unit_var_norm(input_values: List[np.ndarray], input_lengths: List[int]) -> List[np.ndarray]:
+    def zero_mean_unit_var_norm(
```

Comment:
The responsibility of retrieving the correct length from the attention mask should be in this method since `input_values` and `attention_mask` are the well-known inputs to functions in `transformers`